spaceship.zsh

```diff
@@ -405,39 +405,100 @@ spaceship_dir() {
 }
 
 # GIT BRANCH
-# Show current git brunch using git_current_status from Oh-My-Zsh
+# Show current git branch
 spaceship_git_branch() {
   [[ $SPACEHIP_GIT_BRANCH_SHOW == false ]] && return
 
   _is_git || return
 
+  local ref=$(command git symbolic-ref --quiet HEAD 2> /dev/null) ret=$?
+  if [[ $ret != 0 && $ret != 128 ]]; then
+    ref=$(command git rev-parse --short HEAD 2> /dev/null) || return
```

Comment:
Why do you do `ref=$(command git symbolic-ref --quiet HEAD 2> /dev/null)` twice?